internal/v2/business/iam/converter.go

```diff
@@ -42,7 +49,7 @@ func readModelToMembers(readModel *iam_repo.MembersReadModel) []*model.IAMMember
 	return members
 }
 
-func readModelToLabelPolicy(readModel *iam.LabelPolicyReadModel) *model.LabelPolicy {
+func readModelToLabelPolicy(readModel *label.LabelPolicyReadModel) *model.LabelPolicy {
```

Comment:
same as in user ReadModel would be enough if we alias the import to `label_policy` => `label_policy.ReadModel`